students/uss_tomasz/lesson_13_classes/13_1/Circle.py

```diff
@@ -0,0 +1,12 @@
+class Circle:
+    def __init__(self, point, radius):
+        self.point = point
+        self.radius = radius
+
+    def draw(self, drawing_tool):
```

Comment:
Just a side note: passing in a dependency which is later used is known as dependency injection (method injection in this case) design pattern being part of wider inversion of control design pattern.
![image](https://user-images.githubusercontent.com/57070/73593766-ece8f880-4507-11ea-9311-f2fb6af34261.png)